bucket/gifski.json

```diff
@@ -8,7 +8,7 @@
     "extract_dir": "win",
     "bin": "gifski.exe",
     "checkver": {
```

Comment:
```suggestion
    "checkver": "For Windows.*?gifski-([\\d.]+)\\.zip",
```

Reply:
Are you sure? I can't see any tweaks after running `bin\formatjson.ps1`.

Reply:
https://github.com/lukesampson/scoop/blob/3e55a70971c5ff0d035daa54ca5dfab95dfaaa1d/bin/checkver.ps1#L109-L112